app/src/main/java/com/example/vybe/MyVibesAdapter.java

```diff
@@ -68,7 +68,7 @@ public void onBindViewHolder(@NonNull VibeEventHolder holder, int position) {
         holder.vibeImage.setImageResource(vibeEvent.getVibe().getEmoticon());
 
         String itemText = vibeEvent.getVibe().getName();
-        if (vibeEvent.getOwner() != mUsername) {
+        if (vibeEvent.getOwner().equals(mUsername)) {
```

Comment:
Please Verify this part of the code works to ensure .equals works correclty with string comparison and doesn't crash